configure.ac

```diff
@@ -403,11 +404,10 @@ AS_CASE([$host],
   ])
 
 # C preprocessor: we do not rely on AC_PROG_CPP because it returns
-# 'fcc -E' while we really need cpp
-AC_CHECK_TOOL([DIRECT_CPP],[cpp])
-AS_CASE([$host],
-  [*-pc-windows], [CPP="$CC -nologo -EP"],
-  [CPP=$DIRECT_CPP])
+# 'gcc -E' while we really need cpp -- for which host besides *-pc-windows is it necessary?
+#AC_CHECK_TOOL([DIRECT_CPP],[cpp])
```

Comment:
This piece of code was commented out, because it breaks `OCAML_CC_VENDOR` macro (even if you specify `CC=clang` or `CC=xlc`, default C preprocessor can set `OCAML_CC_VENDOR` to `gcc`, if `gcc` is a default compiler).

Reply:
You should be able to eliminate this commenting out when @shindere's https://github.com/shindere/ocaml/commit/3dd1af5f7f8ed45c6f940e2b49fa574b3466808f is merged